9.1/paper.tex

```diff
@@ -206,7 +210,7 @@ \section{Overview}
   element is an enriched version of the $Q_1^d$ element with added bubble
   functions on each edge (in 2d) or face (in 3d). It addresses the fact
   that the $Q_1^d\times Q_0$ combination is not inf-sup stable (requiring a
-  larger velocity space), and that the $Q_2^d\times Q_1$ combination is
+  larger velocity space), and that the $Q_2^d\times Q_0$ combination is
```

Comment:
Hm, I did actually mean the Q2 x Q1 element here. In the Q1 x Q1 combination, the velocity space is too small. In the Q2 x Q1 combination, the velocity space is too large. In the BR x Q1 case, it is just about right.

Or have I completely misunderstood things? I assume that BR x Q1 is stable, right? If you agree with the reasoning above, would you mind reverting the change?

Reply:
I'm thinking in terms of the original paper, which uses the BR1 x Q0 pair because it's a one-degree lower version of Taylor-Hood, except the Q1^d x Q0 pair is not stable. In the [https://doi.org/10.1090/s0025-5718-1985-0771031-7](original paper) part II, they consider the pair where p \in \mathcal{M}_h^{(1)} and u \in X_h, and the definitions of those spaces are at the top of page 72.

I was under the impression that the paragraph I modified was trying to say "Q1^d x Q0 is not stable, and Q2^d x Q0 is not optimal, but BR1 x Q0 is stable and optimal." I could be deriving a totally different meaning from what was intended, which may be the source of my confusion.

Reply:
I actually meant what I said, but what I said may have been wrong. Are you saying that BR1 x Q1 is not stable?

But let's not agonize -- can you just propose a patch that changes the discussion from what you are stating in your second paragraph?

Reply:
I have no reason to believe BR1 x Q1 is stable since it has a larger pressure space compared to the BR1 x Q0 stable pair and a smaller velocity space compared to the Q2^d x Q1 stable pair. It could be stable, but I don't know of any evidence supporting the statement.

You're referring to a patch for "Q1^d x Q0 is not stable, and Q2^d x Q0 is not optimal, but BR1 x Q0 is stable and optimal?" Yes, I can propose a patch for that!